wallet/test/widget_testing/components/wallpaper_widget_test.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:pylons_wallet/pages/detailed_asset_view/widgets/wallpaper_set_screen.dart';
+import 'package:pylons_wallet/components/buttons/custom_paint_button.dart';
+
+import '../../mocks/mock_constants.dart';
+import '../extension/size_extension.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('test case for gestures', (tester) async {
```

Comment:
I dont think testing gesture is a better way. The test should be like 

1. Should show nft image.
2. the set wallpaper button is pressable.